src/renderer/components/hotbar/hotbar-menu.scss

```diff
@@ -12,6 +12,7 @@
   padding-top: 1px;
   width: var(--hotbar-width);
   overflow: hidden;
+  isolation: isolate;
```

Comment:
How does it corresponds with `z-index`?

Reply:
It creates a new [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context) for the hotbar items which means any `z-index` values set for elements **within** hotbar menu don't affect anything outside of it. This is basically needed for showing `.SettingsLayout` properly by covering hotbar items.